packages/components/psammead-timestamp/src/index.jsx

```diff
@@ -18,7 +18,7 @@ const PADDING = `
 
 const StyledTimestamp = styled.time`
   ${props => (props.typographyFunc ? props.typographyFunc(props.script) : '')}
-  color: ${C_CLOUD_DARK};
+  color: ${C_METAL};
```

Comment:
This changes the articles timestamp too, which on zeplin uses `C_CLOUD_DARK` still. Do we need to make this based on a prop? if so, should we raise an issue and do it separately?

Reply:
I actually wanted to ask if the Article was going to be also Metal or we were going to have different ones. Yes, we could raise another PR and I can revert the changes applied to the `Timestamp` files.

Reply:
SGTM :) cheers